capstone/install-wazuh.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Fail if any command fails.
+set -e
+
+# Install needed packages.
+apt-get update
+apt-get install curl apt-transport-https lsb-release gnupg2
+
+# Install GPG key.
+curl -s https://packages.wazuh.com/key/GPG-KEY-WAZUH | apt-key add -
+
+# Add the repository.
+echo "deb https://packages-dev.wazuh.com/staging/apt/ stable main" | tee -a /etc/apt/sources.list.d/wazuh.list
+
+# Update package information.
+apt-get update
+
+# Install the Wazuh manager.
+apt-get install wazuh-manager
+
+
+# Install NodeJS.
+curl -sL https://deb.nodesource.com/setup_10.x | bash -
+apt-get install nodejs
+
+
+# Install Wazuh API.
+apt-get install wazuh-api
+
+
+# Add the Elastic repository and its GPG key.
+apt-get install curl apt-transport-https
+curl -s https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -
+echo "deb https://artifacts.elastic.co/packages/7.x/apt stable main" | tee /etc/apt/sources.list.d/elastic-7.x.list
+apt-get update
+
+
+# Install Filebeat.
+apt-get install filebeat=7.8.1
+
+# Download the Filebeat config file from the Wazuh repository.
+curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/v3.13.1/extensions/filebeat/7.x/filebeat.yml
+
+# Download the alerts template for Elasticsearch.
+curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/v3.13.1/extensions/elasticsearch/7.x/wazuh-template.json
```

Comment:
Use https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json instead. (The one at their master branch instead of the one at their v3.13.1 tag, so we get the latest version)

Reply:
Done